### ISSUE TITLE
Cannot use numpad keys in key shortcuts

### ISSUE THREAD

**Issue by nippur72** [Microsoft/vscode#983]
Zooming in/out with `ctrl +` and `ctrl -` is not possible from the numpad, neither the numpad keys can be referenced specifically so to reconfigure the keymapping.

**Comment by cliffowen** [comment]
In version 1.0 this worked great (on Mac). In 1.1x it is once again broken. This the most compelling feature for me to use VSCode over Sublime and without it I gotta go back to Sublime. Any chance for a fix?

**Comment by alexandrudima** [comment]
@cliffowen What keyboard layout do you have configured?

What happens when you go to Preferences > Keyboard Shortcuts and click on the Define Keybinding and type `cmd + numpad+` , `cmd + numapd-` ?:

Here I am doing it with `ctrl` as I am on a Windows machine:

![keybindings](https://cloud.githubusercontent.com/assets/5047891/15504406/0fbf366e-21bf-11e6-889c-a99e0155a159.gif)

**Comment by cliffowen** [comment]
Again, I am using a Mac.. and using El Capitan (since I did not mention that before). The keyboard layout is ANSI US. I speculate the layout is correct though, as VMWare and Parallels both handle the keyboard properly. Actual keyboard is a Logitech Wave.

Oddly enough, Numpad + and - are fine (numpad_add and numpad_subtract). It's the numbers that are not correct. In the capture window, they report 0, 1, 2... rather than numpad0, numpad1, etc.

Here are a couple of the bindings I had set up and were working on the Mac on 1.0 that no longer work with 1.1.x. My primary goal is identical behavior between VS and VSC on PC and Mac.

    {
        "key": "numpad9",
        "command": "cursorPageUp",
        "when": "editorTextFocus"
    },
    {
        "key": "shift+numpad8",
        "command": "cursorUpSelect",
        "when": "editorTextFocus"
    },
